addons/website/static/src/scss/website.scss

```diff
@@ -1278,6 +1278,95 @@ $-o-carousel-controllers-size: map-get($spacers, 5);
     }
 }
 
+// Single scroll animation for the carousel
+.o_slide_single {
+    --o_flex_value: 50%;
+    /* Set up css variables
+    (Define 1 to 4 cards in the carousel) */
+    .o_with_1_cards {
+        --o_flex_value: 100%;
+    }
+
+    .o_with_2_cards {
+        --o_flex_value: 50%;
+    }
+
+    .o_with_3_cards {
+        --o_flex_value: 33.33%;
+    }
+
+    .o_with_4_cards {
+        --o_flex_value: 25%;
+    }
+
+    @media (min-width: 768px) {
+        .carousel-inner {
+
+            // DISPLAY MULTIPLE CARDS IN A ROW
+
+            display: flex;
+
+            .carousel-item {
+                transition: transform 0.3s ease-in-out;
+            }
+
+            //TODO: Find why this is not working
+            //.carousel-item.active,
+            //.carousel-item.active ~ .carousel-item {
+            //    display: block;
+            //    flex: 0 0 var(--o_flex_value);
+            //    margin-right: 0;
+            //}
+
+            .carousel-item.active,
+            .carousel-item.active + .carousel-item,
+            .carousel-item.active + .carousel-item + .carousel-item,
+            .carousel-item.active + .carousel-item + .carousel-item + .carousel-item,
+            .carousel-item.active + .carousel-item + .carousel-item + .carousel-item + .carousel-item {
+                display: block;
+                flex: 0 0 var(--o_flex_value);
+                margin-right: 0;
+            }
+
+            /*   SLIDE LEFT ANIMATION  */
+            .carousel-item-start.active,
+            .carousel-item-start:not(.active),
+            .carousel-item-start:not(.active) ~ .carousel-item {
+                transform: translateX(-100%);
+            }
+
+            .carousel-item.active:not(.carousel-item-start):not(.carousel-item-end),
+            .carousel-item.active:not(.carousel-item-start):not(.carousel-item-end) ~ .carousel-item {
+                transition: none;
+                margin-right: initial;
+            }
+
+            .carousel-item-next:not(.carousel-item-start) {
+                transform: translateX(0%);
+            }
+
+            /*   SLIDE RIGHT ANIMATION  */
+            .carousel-item-end:not(.active),
+            .carousel-item-prev:not(.active),
+            .carousel-item-prev:not(.active) ~ .carousel-item {
+                display: block;
+                flex: 0 0 var(--o_flex_value);
+                margin-right: 0;
+            }
+
+            //.active.carousel-item-end {
+            //  transform: translateX(0%);
+            //}
+            .active.carousel-item-end + .carousel-item-prev.carousel-item-end,
+            .carousel-item-prev.carousel-item-end ~ .carousel-item {
+                right: var(--o_flex_value);
+                transform: translateX(100%);
+                display: block;
+            }
+        }
+    }
```

Comment:
These are all the necessary CSS modifications to 'cancel' the original full-slide animation and enable the single-scroll mode. It works, except when you press the 'prev' button when the number of elements fetched is one more than the number of elements displayed in the carousel. In that case, the animation behaves strangely.